Sample/MVC/TicTacToe/Sources/AppDelegate.swift

```diff
@@ -23,7 +23,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        registerDependencyProviderFactories()
+        registerProviderFactories()
```

Comment:
All the MVC changes, assume just because we hadn't updated this one recently ? I guess we were only looking at the other one.